### PR TITLE
Highlight ongoing sessions

### PR DESCRIPTION
**Pull request by ytRino**
## Issue
- #161

## Overview (Required)
- Highlight ongoing sessions. It checks every session with `new Date()` but I don't have any good pattern for this and implemented such roughly as-is.
- It seems `Session.Type#toString()` is overriden just for `equals` check.  
If `Session.type` is `Type type`, `Session#isBreak()` will be `type == Type.BREAK`.
idk why `Session.type` is not `Type` (im not familiar with orma, adapting enum brings complexity?) so just make `Type` to have its own equality method using `name().equalsIgnoreCase()`. I will revert this change as you like (eg. json should strictly declare lowercase for `type` key. this change will be breaking).

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

**Comment by konifar** [comment]
> why Session.type is not Type (im not familiar with orma, adapting enum brings complexity?)

As you said, it's better to change `type` to enum since orma supports enum.
Anyway, I'm for your change.

File: app/src/main/res/drawable-v21/clickable_purple.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+        android:color="@color/black_alpha_12"
+        >
+
+    <item
+            android:id="@android:id/mask"
+            android:drawable="@android:color/white"
+            />
+
+    <item android:drawable="@color/purple_alpha_15" />
```
Comment (konifar) [inline app/src/main/res/drawable-v21/clickable_purple.xml]: I specified this color, but maybe the different color is better.
It's another issue, so no need to change it on this PR! 👍

**Comment by konifar** [comment]
LGTM! Thanks! 💯

File: app/src/main/java/io/github/droidkaigi/confsched2017/model/Session.java [hunk 2]
```diff
@@ -87,26 +86,25 @@
     private enum Type {
         CEREMONY, SESSION, BREAK;
 
-        @Override
-        public String toString() {
-            return super.toString().toLowerCase();
+        boolean matches(String type) {
+            return name().equalsIgnoreCase(type);
```
Comment (jmatsu) [inline app/src/main/java/io/github/droidkaigi/confsched2017/model/Session.java]: `equalsIgnoreCase` depends on device's Locale, so can you use `name().toLowerCase(Locale.US).equals(type.toLowerCase(Locale.US))`?

Comment (jmatsu) [inline app/src/main/java/io/github/droidkaigi/confsched2017/model/Session.java]: Sorry, my review is late. Please don't mind.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/model/Session.java]: Oh, sorry... Your review makes sense! Let's fix it on another PR 💪

Comment (ytRino) [inline app/src/main/java/io/github/droidkaigi/confsched2017/model/Session.java]: wow 😮 interesting.
I didn't know that and take a quick search, then javadoc and some blog posts are pointing 
that `equalsIgnoreCase()` is locale *in*dependent.
https://developer.android.com/reference/java/lang/String.html#equalsIgnoreCase(java.lang.String)
http://yuki312.blogspot.jp/2013/05/androidtolowercasetouppercase.html
I'm not tested myself but if you have some clarification, plz let me know. thx!

**Pull request merged**